tests/test_free_energy.py

```diff
@@ -358,6 +360,10 @@ def low_dim_f(perturb: float) -> float:
                 abs_err = check_grad(low_dim_f, grad(low_dim_f), perturb, epsilon=1e-4)
                 assert abs_err < 1e-3
 
+        # grad w.r.t. box shouldn't be allowed
+        with pytest.raises(RuntimeError):
+            _ = grad(U, argnums=2)(coords, sys_params, box, lam)
```

Comment:
nit: worth checking the error message to make sure we're getting  the expected `RuntimeError`?

Reply:
Will fix!

Reply:
Added in https://github.com/proteneer/timemachine/pull/707/commits/048e5d7fea60578063a1217686ba1493dae8aae2 !